app/models/etsource/base.rb

```diff
@@ -116,6 +116,12 @@ def get_latest_import_sha
       nil
     end
 
+    def last_updated_at(folder)
+      unless Rails.env.test?
+        git.log(1).object(folder).map(&:date).first
```

Comment:
I think this might not be quite correct: production and staging environments might be running a different ETSource version than is checked out using Git.

`config/config.yml` files have two options: `etsource_working_copy` and `etsource_export`. In development, these both point to the same directory. This means our local ETEngines use ETSource directly.

In staging and production, `etsource_export` has a different path. When someone checks out an ETS version on the [ETSource page](http://beta.et-engine.com/etsource/commits), the contents of that commit are copied into a different directory (`etsource_export`). ETEngine reads data from *that* directory, not the Git clone (`etsource_working_copy`). The live paths are something like:

* `etsource_export`: /var/www/etengine/current/tmp/etsource
* `etsource_working_directory`: /var/www/etsource

It's possible therefore that ETEngine may be using an older ETSource commit:

![screen shot 2016-01-28 at 11 15 07](https://cloud.githubusercontent.com/assets/4383/12642883/81bac924-c5b0-11e5-8695-690f008b9f8e.png)

I think you could work around this by supplying the current in-use Git revision:

```ruby
git.gcommit(get_latest_import_sha)
  .log(1).object(path).map(&:date).first
```